.gitignore

```diff
@@ -117,3 +117,6 @@ prime/
 
 # Manpage
 /man
+
+# IntelliJ Gateway
+.uuid
```

Comment:
can you add this close to line 10? next to the other intellij line.

Reply:
done